docs/language/search-syntax/README.md

```diff
@@ -547,153 +466,129 @@ If you enter multiple [value match](#value-match) or
-...
+{School:"C. A. Jacobs Intermediate",District:"Dixon Unified",City:"Dixon",County:"Solano",Zip:"95620-3209",Latitude:38.446472,Longitude:-121.83631,Magnet:false,OpenDate:1980-07-01T00:00:00Z,ClosedDate:null (time),Phone:"(707) 693-6350",StatusType:"Active",Website:"www.dixonusd.org"} (=school)
+{School:"Dixon Adult",District:"Dixon Unified",City:"Dixon",County:"Solano",Zip:"95620",Latitude:38.444818,Longitude:-121.82287,Magnet:null,OpenDate:1996-09-09T00:00:00Z,ClosedDate:2016-06-30T00:00:00Z,Phone:null,StatusType:"Closed",Website:null} (school)
+{School:"Dixon Community Day",District:"Dixon Unified",City:"Dixon",County:"Solano",Zip:"95620",Latitude:38.44755,Longitude:-121.82001,Magnet:false,OpenDate:2003-08-23T00:00:00Z,ClosedDate:null,Phone:"(707) 693-6340",StatusType:"Active",Website:"www.dixonusd.org"} (school)
+{School:"Dixon High",District:"Dixon Unified",City:"Dixon",County:"Solano",Zip:"95620-9301",Latitude:38.436088,Longitude:-121.81672,Magnet:false,OpenDate:1980-07-01T00:00:00Z,ClosedDate:null,Phone:"(707) 693-6330",StatusType:"Active",Website:null} (school)
+{School:"Dixon Montessori Charter",District:"Dixon Unified",City:"Dixon",County:"Solano",Zip:"95620-2702",Latitude:38.447984,Longitude:-121.83186,Magnet:false,OpenDate:2010-08-11T00:00:00Z,ClosedDate:null,Phone:"(707) 678-8953",StatusType:"Active",Website:"www.dixonmontessori.org"} (school)
+{School:"Dixon Unified Alter. Educ.",District:"Dixon Unified",City:"Dixon",County:"Solano",Zip:"95620",Latitude:null,Longitude:null,Magnet:null,OpenDate:1993-08-26T00:00:00Z,ClosedDate:1994-06-30T00:00:00Z,Phone:null,StatusType:"Closed",Website:null} (school)
+{School:"Maine Prairie High (Continuation)",District:"Dixon Unified",City:"Dixon",County:"Solano",Zip:"95620-3019",Latitude:38.447549,Longitude:-121.81986,Magnet:false,OpenDate:1980-07-01T00:00:00Z,ClosedDate:null,Phone:"(707) 693-6340",StatusType:"Active",Website:null} (school)
+{School:null,District:"Dixon Unified",City:"Dixon",County:"Solano",Zip:"95620-3447",Latitude:38.44468,Longitude:-121.82249,Magnet:null,OpenDate:null,ClosedDate:null,Phone:"(707) 693-6300",StatusType:"Active",Website:"www.dixonusd.org"} (school)
 ```
 
 > **Note:** `!` can also be used as alternative shorthand for `not`.
-
-        zq -f table '! _path matches /conn|dns|files|ssl|x509|http|weird/' *.log.gz
+> ```
+> ! elementary District=="Dixon Unified"
+> ```
 
 ### Parentheses & Order of Evaluation
 
 Unless wrapped in parentheses, a search is evaluated in _left-to-right order_.
+Terms wrapped in parentheses will be evaluated _first_, overriding the default
+left-to-right evaluation.
 
-For example, the following search leverages the implicit boolean `and` to find
-all `smb_mapping` records in which the `share_type` field is set to a value
-other than `DISK`.
+For example, we've noticed there's some test score records that have `null`
```

Comment:
```suggestion
For example, we've noticed there're some test score records that have `null`
```

Reply:
@nwt: Good catch. I've now gone with `there are` over `there're`. I know `there're` is valid, but it always looks awkward to me.